assignment/index.html

```diff
@@ -58,25 +58,49 @@
 
       /* =====================
 
-      Start code
+      Start c
 
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
-
-      /* =====================
-
-      End code
-
-      ===================== */
-
-
-      jsonToCsv(healthCenters);
-      addMarkers(map);
-    </script>
-    <!--Your code ends here-->
-  </body>
-</html>
+        let HospitalLocation = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
```

Comment:
It'd be good to not have to type out each of these keys manually.

Take a look how at this: `Object.keys(healthCenters[0])` Does all the work for you!

Reply:
This makes it easier. Thanks